scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -4123,6 +4123,16 @@ def run(self):
                 self.__report_container_metrics,
             )
         )
+
+        server = scalyr_util.get_web_url_from_upload_url(
+            self._global_config.scalyr_server
+        )
+        global_log.info(
+            "View data from this agent at: %s/events?filter=$serverHost%%3D%%27%s%%27&log=%%2Fvar%%2Flog%%2Fscalyr-agent-2%%2Fagent.log"
```

Comment:
Let's change the leading string to "View the log for this agent at:"

Hmmm... there is a bug in this code in that if `serverHost` has characters that need to be escaped in a string or as  a URL parameter, we won't handle it correctly.  You aren't introducing this bug.. it was already there to begin with.

I don't have a quick fix for this, so maybe add a TODO saying "Handle escaping appropriate characters in serverHost string".

Reply:
Sadly the `urllib` way of encoding url params changed between python 2 and 3, or else it would have been trivial.

Reply:
Actually, `six` provides a compatibility layer for that, I believe.  Check out this sample code: https://github.com/scalyr/scalyr-agent-2/blob/master/tests/smoke_tests/request.py#L48